### PR TITLE
test update to sizing doc

### PR DESCRIPTION
**Pull request by tzach**
Goal if this PR is to test preview

### CODE REVIEW

**Comment by tzach** [comment]
@dgarcia360 I do not see the PR review, can you please take a look?

**Comment by raoufchebri** [comment]
Not working for me

File: docs/sizing.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ model for this project that has the following characteristics:
     measurement * 60 minutes / hour * 24 hours * 30 days =
     = 43,200 rows / partition
 
-#### Considerations
+#### Considerations and notes
```
Comment (tzach) [inline docs/sizing.md:16]:
```suggestion
#### Considerations and note
```

**Comment by tzach** [comment]
test works, preview works, deleting the PR

**Pull request closed without merging**